src/components/DropdownSelect.js

```diff
@@ -68,7 +74,11 @@ export const DropdownSelect = ({
   }, [selectedData]);
 
   return (
-    <View>
+    <View
+      accessible
+      accessibilityLabel={`(${label}) (${multipleSelect ? selectedMultipleValues : selectedValue}) 
```

Comment:
please order the props `accessible` and `accessibilityLabel` always in the same way

Reply:
I can delete the brackets in the label because brackets are used in the rest of the sentence. 👍

https://github.com/smart-village-solutions/smart-village-app-app/commit/0a6c8ce7fa5ed299446fb3ab7a910a08b673a271

Reply:
> please order the props accessible and accessibilityLabel always in the same way

you missed that, right?

Reply:
I think it's done now. 🤷‍♂️

https://github.com/smart-village-solutions/smart-village-app-app/commit/2b70e4c58f4da0633c16f4fb6de915210e644970